aten/src/ATen/native/sparse/SparseCsrTensorMath.h

```diff
@@ -59,6 +61,64 @@ inline void _check_dim(const Tensor& self, int64_t target_dim, c10::string_view
       " instead.");
 }
 
+inline SPMM_REDUCE_OP get_operator_enum(const c10::string_view reduce) {
+  if (reduce == "sum") {
+    return SPMM_SUM;
+  } else if (reduce == "mean") {
+    return SPMM_MEAN;
+  } else if (reduce == "max") {
+    return SPMM_MAX;
+  } else if (reduce == "min") {
```

Comment:
```suggestion
  } else if (reduce == "amax") {
    return SPMM_MAX;
  } else if (reduce == "amin") {
```

Reply:
OK, i will change the reduce flags to align with other reduce ops

Reply:
Fixed!